Cargo.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["frb_codegen", "frb_macros", "something"]
```

Comment:
btw would you mind creating a separate pr about "add cargo workspace" and I will merge that. o/w this pr is doing two things into one pr

Reply:
this wasn't supposed to be in the final PR, just so my rust-analyzer work

Reply:
This can be in the final PR - cargo workspaces are good practice :)

Reply:
Indeed, my rust-analyzer won't even work without the workspace. But do you want it to be in a separate PR?

Reply:
Yes, making it a separate PR will be much cleaner. But if you want to merge in one pr it is also acceptable